_commands/toys/sl.md

```diff
@@ -0,0 +1,49 @@
+---
+---
+
+sl
+-------
+
+Prints a train whenever you mistype ls as sl.
+
+~~~ bash
+$ sl
+~~~
+
+<!--more-->
+
+~~~bash
+$ sl
+     ====        ________                ___________
+  _D _|  |_______/        \__I_I_____===__|_________|
+   |(_)---  |   H\________/ |   |        =|___ ___|      _________________ 
+   /     |  |   H  |  |     |   |         ||_| |_||     _|                \_____A
+  |      |  |   H  |__--------------------| [___] |   =|                        |
+  | ________|___H__/__|_____/[][]~\_______|       |   -|                        |
+  |/ |   |-----------I_____I [][] []  D   |=======|____|________________________|_
+__/ =| o |=-~~\  /~~\  /~~\  /~~\ ____Y___________|__|__________________________|_
+ |/-=|___||    ||    ||    ||    |_____/~\___/          |_D__D__D_|  |_D__D__D_|
+  \_/      \__/  \__/  \__/  \__/      \_/               \_/   \_/    \_/   \_/
+~~~
+
+
+It is like the example above, but better.
+
+### How to install sl
+~~~bash
+$ sudo apt-get install sl
+~~~
+
+### Special flags
+-a <br>
+ the train flies
+
+-l <br>
+ a small train
+
+-F <br>
+ a flying train
+
+-e <br>
+ allows ctrl+c
+
```

Comment:
We want to avoid using raw html tags in the markdown files. It turns out that hitting enter/return twice will give you one line of space between the lines-- but I have a better idea. We can just make this into a list, like this ...

~~~markdown
* `-a` makes the train fly
* `-l` animates a smaller train ...
...
(etc.)
~~~

... which will come out like this ...

* `-a` makes the train fly
* `-l` animates a smaller train ...